guides/common/modules/proc_configuring-repositories-proxy.adoc

```diff
@@ -0,0 +1,133 @@
+[id="configuring-repositories-proxy_{context}"]
+
+= Configuring Repositories
+
+ifeval::["{build}" == "foreman"]
+This procedure is only for Katello plug-in and {RHEL}-based operating system users.
+endif::[]
+
+Use this procedure to enable the repositories that are required to install {ProductName}.
+
+.Procedure
+To configure the required repositories, complete the following steps:
+
+. Disable all repositories:
++
+[options="nowrap"]
+----
+# subscription-manager repos --disable "*"
+----
++
+. Enable the following repositories:
+ifeval::["{build}" == "foreman"]
++
```

Comment:
This needs to include the [options="nowrap" subs="+quotes,attributes"] line or the RepoRHEL7Server attribute won't work.